src/api/cpp/cvc5.h

```diff
@@ -4102,49 +4131,59 @@ class CVC5_EXPORT Solver
 
   /**
    * Do quantifier elimination.
+   *
    * SMT-LIB:
-   * \verbatim
-   * ( get-qe <q> )
-   * \endverbatim
+   *
+   *     ( get-qe <q> )
+   *
    * Requires a logic that supports quantifier elimination. Currently, the only
    * logics supported by quantifier elimination is LRA and LIA.
-   * @param q a quantified formula of the form:
-   *   Q x1...xn. P( x1...xn, y1...yn )
-   * where P( x1...xn, y1...yn ) is a quantifier-free formula
-   * @return a formula ret such that, given the current set of formulas A
-   * asserted to this solver:
-   *   - ( A ^ q ) and ( A ^ ret ) are equivalent
-   *   - ret is quantifier-free formula containing only free variables in
-   *     y1...yn.
+   * @param q a quantified formula of the form
+   *          @f$Q x_1...x_n. P( x_1...x_n, y_1...y_n )@f$ where
```

Comment:
Can we handle quantifier alternations here? @ajreynol

Reply:
Yes.

FYI, the documentation should be updated to convey that LRA and LIA are the only ones where QE is complete, although I believe we should support any theory in an incomplete way here.

Reply:
I added a sentence.

Reply:
If we support quantifier alternations, we should update to something like
`@f$ Q_1 x_1...x_n Q_2 x_{n+1}...x_{m}. P( x_1...x_m, y_1...y_n)@f$`. Also, we should not use `n` for both x and y, they do not need to be the same amount of `x`s and `y`s.